addons/sale_loyalty/models/sale_order.py

```diff
@@ -1303,17 +1303,18 @@ def _try_apply_program(self, program, coupon=None):
             return {'error': _('This program is already applied to this order.'), 'already_applied': True}
         elif program.reward_ids:
             global_reward = program.reward_ids.filtered('is_global_discount')
+            max_global_reward = global_reward and max(global_reward, key=lambda r: r.discount)
```

Comment:
As previously mentioned, this is not the best direction, since 11% will be always seen as less than 15$, which will not always be the case.